README.md

```diff
@@ -40,7 +40,7 @@ Bond C++ library requires some C++11 features (currently limited to those
 supported bv Visual C++ 2013); a C++11 compiler is required. Additionally,
 to build Bond you will need CMake (3.1+),
 [Haskell Stack](https://docs.haskellstack.org/en/stable/README/#how-to-install)
-(1.5.1+) and Boost (1.58+).
+(1.5.1+) and Boost (1.61+).
```

Comment:
Update the minimum version check in [cmake/config.cmake](https://github.com/Microsoft/bond/blob/master/cmake/Config.cmake#L126)?